packages/terra-notification-dialog/src/terra-dev-site/doc/notification-dialog/NotificationDialog.1.doc.mdx

```diff
@@ -0,0 +1,68 @@
+import { Badge } from 'terra-notification-dialog/package.json?dev-site-package';
+
+import NotificationDialogPropsTable from 'terra-notification-dialog/src/NotificationDialog?dev-site-props-table';
+
+import CompleteNotificationDialog from '../example/CompleteNotificationDialog?dev-site-example';
+import CompleteNotificationDialogWithLongMessage from '../example/CompleteNotificationDialogWithLongMessage?dev-site-example';
+import ContentNotificationDialog from '../example/ContentNotificationDialog?dev-site-example';
+import NoVariantNotificationDialog from '../example/NoVariantNotificationDialog?dev-site-example';
+import NotificationDialogWithLongText from '../example/NotificationDialogWithLongText?dev-site-example';
+import NotificationDialogOnModalManager from '../example/NotificationDialogOnModalManager?dev-site-example';
+import RejectFirstNotificationDialog from '../example/RejectFirstNotificationDialog?dev-site-example';
+import EmphasizedRejectNotificationDialog from '../example/EmphasizedRejectNotificationDialog?dev-site-example';
+
+<Badge />
+
+# Terra NotificationDialog
+
+`terra-notification-dialog` is a notification dialog component built over the `terra-abstract-modal`. It has the highest z-index of 9001. It is a common component to be used for confirmation/acceptance criteria style dialogs.
+<br />**Note:** Use at least one of `acceptAction`, `rejectAction`, `primaryAction` or `secondaryAction` props for proper functioning of the `Notification Dialog` and avoid getting a warning.
```

Comment:
"Note:" and "deprecated" are not bold on the [dev-site](https://terra-framework-deploy-pr-1059.herokuapp.com/components/terra-notification-dialog/notification-dialog/notification-dialog#terra-notificationdialog)

Reply:
Fixed 8a58b84